src/mod/externals/FlagWork_Enums.h

```diff
@@ -1770,7 +1770,12 @@ enum class FlagWork_Flag : int32_t {
     FLAG_TRAINER_DOUBLE = 2196,
     FLAG_DISABLE_LEVEL_CAP = 2197,
     FLAG_DISABLE_EXP_SHARE = 2198,
-    FLAG_AFFECTION_ENABLED = 2199,
+    FLAG_DISABLE_AFFECTION = 2199,
+    FLAG_DISABLE_VISIBLE_SHINY_EGGS = 2200,
+    FLAG_TEMP_DISABLE_LEVEL_CAP = 2201,
+    FLAG_TEMP_DISABLE_EXP_SHARE = 2202,
+    FLAG_TEMP_DISABLE_AFFECTION = 2203,
+    FLAG_TEMP_DISABLE_VISIBLE_SHINY_EGGS = 2204,
```

Comment:
Flag 2200 is already used in scripts (we haven't put stuff used in scripts into this enum yet)
You should be able to use 2201-2205, just make sure to double-check in the [Romhack](https://github.com/TeamLumi/Romhack) repo for any future flags you might want to use. Extended save starts at 4001 if you wanna be certain they're unused